xdsl/dialects/riscv.py

```diff
@@ -434,6 +453,49 @@ class BgeuOp(RsRsOffOperation):
     name = "riscv.bgeu"
 
 
+# RV32I/RV64I: 2.6 Load and Store Instructions
+
+
+@irdl_op_definition
+class LBOp(RdRsImmOperation):
```

Comment:
We've been migrating the Op names to a CamelCase where the whole instruction name is a single identifier, could you please change this operation and below?
```suggestion
class LbOp(RdRsImmOperation):
```